internal/pkg/helm/values_inspector.go

```diff
@@ -5,11 +5,22 @@ import (
 	"github.com/sirupsen/logrus"
 	"k8s.io/helm/pkg/chartutil"
 	"reflect"
+	"regexp"
 	"strings"
 )
 
 const goTemplateMemberAccessOperator = "."
 
+//LogHelmReport - Collect Log data and print
```

Comment:
Is this new functionality since the initial patch?  It is generally better to introduce new things in subsequent commits as it gets hard to review.  Additionally, this is only utilized by part of the code, why not stick to the standard logrus interface?  logrus is what is used by everything else.  Without adopting this across the code base, this is going to have only for-loop related data.

Reply:
I am blocked with PR reviews getting delayed. Can we please let this go together .This is just one struct I introduced to capture what FOR loop is doing .